rpm/build/SOURCES/jenkins.sysconfig.in

```diff
@@ -38,13 +38,34 @@ JENKINS_USER="@@ARTIFACTNAME@@"
 #
 #JENKINS_INSTALL_SKIP_CHOWN="false"
 
-## Type: string
-## Default:     "-Djava.awt.headless=true"
+## Type:        string
+## Default:     "-server -XX:+AlwaysPreTouch -XX:+ParallelRefProcEnabled"
+## ServiceRestart: @@ARTIFACTNAME@@
+# Sensible defaults, server is normally default but we ensure it is on
+# AlwaysPreTouch prezeroes allocated heap pages at startup, which improves runtime performance at slight cost to startup time
+# ParallelRefProcEnabled enables use of concurrent threads to process references, which Jenkins uses heavily for caching
+BASE_JAVA_ARGS=' -server -XX:+AlwaysPreTouch -XX:+ParallelRefProcEnabled '
```

Comment:
This is unnecessarily surrounded by spaces.  You already surround the variable with spaces in `JENKINS_JAVA_OPTIONS`.

In other parts of this PR the same feedback applies.  However, I'll only comment this once.